test/javascripts/acceptance/tag-autocomplete-test.js.es6

```diff
@@ -0,0 +1,37 @@
+import { acceptance } from "helpers/qunit-helpers";
+import { search } from 'discourse/plugins/discourse-tagging/lib/category-tag-search';
+acceptance("Tag Autocomplete", {
+  loggedIn: true,
+  setup() {
+    const response = (object) => {
+      return [
+        200,
+        {"Content-Type": "application/json"},
+        object
+      ];
+    }
+
+    server.get('/tags/filter/search', () => {
+      return response({ results: [{ text: 'monkey', count: 1 }] });
+    });
+
+    server.get('/category_hashtags/check', () => {
+      return response({ valid: [] });
+    })
+
+    server.get('/tags/check', () => {
+      return response({ valid: [{ value: 'monkey', url: '/tags/monkey' }] })
+    });
+  }
+});
+
+test("tag is cooked properly", () => {
+  visit("/");
+  click('#create-topic');
+
+  fillIn('.d-editor-input', "this is a tag hashtag #monkey::tag");
+  andThen(() => {
+    // TODO: Test that the autocomplete shows
```

Comment:
I spent a few hours trying to get the autocomplete to show but it doesn't show even if I trigger a `keypress`.

I'm planning to remove the `keypress` event for autocomplete since it doesn't work at all on Android.